### ISSUE TITLE
“Visit add-ons blog” is not linked

### ISSUE THREAD

**Issue by acornestean** [mozilla/extension-workshop#223]
### Describe the problem and steps to reproduce it:
“Visit add-ons blog” at the bottom of the EW page is not linked.

Steps to reproduce:
1. Load the EW page
2. Navigate to the bottom of the page until reaching the “Visit add-ons blog” text
3. Attempt to click on it
4. Notice that “Visit add-ons blog” is not linked

### What happened?
“Visit add-ons blog” is not linked.

### What did you expect to happen?
“Visit add-ons blog” should be linked and redirect the user to the corresponding page upon clicking it.

### Anything else we should know?
The issue occurs on macOS 10.13.6 and Windows 10 Pro 64-bit.

Please note that the issue only occurs when the browser window is maximized. When making the browser window smaller (so as to force the page's Hamburger Menu to be displayed) the string is linked and redirects as expected. For further information, see the second attachment.

![Visit add-ons blog not linked](https://user-images.githubusercontent.com/50236075/61871581-d96e2c80-aee9-11e9-9cb5-6e8f07bd9cf1.gif)

![Visit add-ons blog linked when minimized](https://user-images.githubusercontent.com/50236075/61872547-4a164880-aeec-11e9-8a57-4d1b55d6da2d.gif)

**Comment by muffinresearch** [comment]
To fix this `pointer-events: none` is needed on `.bottom-of-page`.

**Comment by AlexandraMoga** [comment]
Verified fixed:

![2019-07-29_09-46-25](https://user-images.githubusercontent.com/31961530/62027247-dfb02180-b1e5-11e9-849a-e2552dde7a85.gif)
